# Safe depth altitude parameters
max_depth: 4.0
min_altitude: 1.0
min_altitude_starts_at_depth: 0.5
